IDEAS/BoundaryConditions/Interfaces/PartialSimInfoManager.mo

```diff
@@ -1,48 +1,67 @@
-  parameter Boolean computeConservationOfEnergy = false
+
+
+  parameter SI.Angle incAndAziInBus[:,:] = {{IDEAS.Types.Tilt.Ceiling,0},{IDEAS.Types.Tilt.Wall,IDEAS.Types.Azimuth.S},
+                         {IDEAS.Types.Tilt.Wall,IDEAS.Types.Azimuth.W},{IDEAS.Types.Tilt.Wall,IDEAS.Types.Azimuth.N},{IDEAS.Types.Tilt.Wall,IDEAS.Types.Azimuth.E}}
+                        "Combination of inclination and azimuth which are pre-computed and added to solBus." annotation(Dialog(tab="Incidence angles"));
+
+  parameter Boolean computeConservationOfEnergy=false
     "Add equations for verifying conservation of energy"
-    annotation(Evaluate=true,Dialog(tab="Conservation of energy"));
-  parameter Boolean strictConservationOfEnergy = false
+    annotation (Evaluate=true, Dialog(tab="Conservation of energy"));
+  parameter Boolean strictConservationOfEnergy=false
     "This adds an assert statement to make sure that energy is conserved"
-    annotation(Evaluate=true,Dialog(tab="Conservation of energy", enable = computeConservationOfEnergy));
-  parameter Boolean openSystemConservationOfEnergy = false
-    "Compute conservation of energy for open system"
-    annotation(Evaluate=true,Dialog(tab="Conservation of energy", enable = computeConservationOfEnergy));
+    annotation (Evaluate=true, Dialog(tab="Conservation of energy", enable=
+          computeConservationOfEnergy));
+  parameter Boolean openSystemConservationOfEnergy=false
+    "Compute conservation of energy for open system" annotation (Evaluate=true,
+      Dialog(tab="Conservation of energy", enable=computeConservationOfEnergy));
 
-  parameter Modelica.SIunits.Energy Emax = 1
-    "Error bound for violation of conservation of energy"
-    annotation(Evaluate=true,Dialog(tab="Conservation of energy", enable = strictConservationOfEnergy));
+  parameter Boolean linearise=false "Linearises building model equations"
+    annotation (Dialog(tab="Linearisation"));
+  parameter Boolean createOutputs = false
+    "Creates output connections when linearising windows"
+    annotation(Dialog(tab="Linearisation"));
+  parameter Boolean outputAngles=not linearise
+    "Output angles in weaBus. Set to false when linearising" annotation(Dialog(tab="Linearisation"));
+  parameter Boolean linIntCon= false
+    "= true, if interior convective heat transfer should be linearised"
+    annotation (Dialog(tab="Linearisation", group="Convection"));
+  parameter Boolean linExtCon= false
+    "= true, if exterior convective heat transfer should be linearised (uses average wind speed)"
+    annotation (Dialog(tab="Linearisation", group="Convection"));
+  parameter Boolean linIntRad= true
+    "= true, if interior radiative heat transfer should be linearised"
+    annotation (Dialog(tab="Linearisation", group="Radiation"));
+  parameter Boolean linExtRad= true
+    "= true, if exterior radiative heat transfer should be linearised"
+    annotation (Dialog(tab="Linearisation", group="Radiation"));
```

Comment:
@Mathadon @damienpicard For me these defaults are fine, i.e. convection non-linear and radiation linearised. We should however clearly describe this somewhere.